src/main/java/labs/introtoprogramming/lab5/object/Parser/ParseObjFile.java

```diff
@@ -0,0 +1,155 @@
+package labs.introtoprogramming.lab5.object.Parser;
+
+import labs.introtoprogramming.lab5.geometry.Vector3;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.INFO;
+import static java.util.logging.Level.WARNING;
+
+public class ParseObjFile {
+    private Logger log = Logger.getLogger(ParseObjFile.class.getName());
+
+    /**
+     * String indicators of parameter in .obj file.
+     */
+
+    private final static String OBJ_VERTEX_NORMAL = "vn";
+    private final static String OBJ_VERTEX = "v";
+    private final static String OBJ_FACE = "f";
+
+    private ArrayList<Vector3> verticesNormals = new ArrayList<>();
```

Comment:
Better change `ArrayList<Vector3>` to `List<Vector3>`, as list implementation may change over time